src/DelayBucket.h

```diff
@@ -19,20 +19,22 @@ class DelayBucket
 {
 
 public:
+    typedef int BucketLevel;
```

Comment:
Does this naming change change any types? Ideally, I do not want this PR to become responsible for the underlying level type because that type should not be `int` but changing it correctly is a difficult problem that deserves its own PR. If this naming change preserves all existing types, then let's undo it.

Reply:
This type was used in DelayBucket::update() to avoid specifying 'int' explicitly, but we don't need it anymore after changing into SetToNaturalSumOrMax().